collections/landsat-1-5-mss-l1.yaml

```diff
@@ -114,6 +114,6 @@ CubeDimensions:
       - SAA
       - SZA
       - dataMask
-
+sci:citation: Landsat 1-5 product courtesy of the U.S. Geological Survey, processed with Sentinel Hub
```

Comment:
Is the last part ok / was it agreed somewhere?

I only find `XYZ product courtesy of the U.S. Geological Survey` from the links in the description.

Reply:
I altered the statement we have on [our homepage](https://www.sentinel-hub.com/faq/how-do-i-cite-sentinel-playground-eo-browser-or-other-web-pages-applications/) for Sentinel ("Modified Copernicus Sentinel data [Year]/Sentinel Hub") and think it is okay like that.